Examples/Scripts/Python/common.py

```diff
@@ -44,48 +44,23 @@ def addPythia8(
     vertexStddev: acts.Vector4 = acts.Vector4(0, 0, 0, 0),
     vertexMean: acts.Vector4 = acts.Vector4(0, 0, 0, 0),
 ):
-    vertexGenerator = acts.examples.GaussianVertexGenerator(
-        stddev=vertexStddev, mean=vertexMean
+    """This function steers the particle generation using Pythia8
+
+    NB. this version is included here only for compatibility. Please use pythia8.addPythia8 instead.
```

Comment:
Should we do a deprecation warning then? Or maybe just remove this one?

In any case, if it's just a forward, one could use `*args, **kwargs` to forward instead of listing all the defaults again here.

Reply:
I think we can remove this one, but lets do that in a separate PR.

Unfortunately a few of the kwargs are a bit different (`beam` and `vtxGen`) to fit with our new style.

Reply:
Ok